pkg/terraform/packrd/packed-packr.go

```diff
@@ -0,0 +1,30 @@
+// +build !skippackr
+// Code generated by github.com/gobuffalo/packr/v2. DO NOT EDIT.
+
+// You can use the "packr2 clean" command to clean up this,
+// and any other packr generated files.
+package packrd
```

Comment:
We have switched to not checking in any `*-packr.go` files and putting that in the .gitignore file to enforce it. Again fine to fix here or in a follow-up PR.